content/en/os/1.13.x/login/regaining-access/index.markdown

```diff
@@ -0,0 +1,90 @@
++++
+title = "Regaining Access to a Bottlerocket Node After Disabling the Control and Admin Containers"
+type = "docs"
++++
+
+## Introduction
+
+The standard way to access a shell on a Bottlerocket node is to use either the admin container or the control container.
+In some cases, both the admin and control containers may be disabled.
+There is a way to regain access to a Bottlerocket node in such a scenario.
+
+## Solution Description
+
+In general, the solution is to mount the [API client](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/apiclient/README.md) and API socket into a container on the Bottlerocket node and use the API client to re-enable the admin container, control container, or both.
+
+## Solution Steps
+
+There are different ways to implement the solution depending on the container orchestrator used to manage the Bottlerocket nodes.
```

Comment:
This sentence isn't really true because is (currently) only one way to implement the solution and that way exists in k8s. I would write something along those lines here.

Reply:
"Currently, this solution only works on Kubernetes. Bottlerocket does not _yet_ support `ecs exec`, so this solution does not yet work on ECS." 

(and removing the "Steps to Regain Access on ECS" section below)

Thoughts?

Reply:
I think I'd just remove this entire section and make kubernetes and ecs sections both heading 2.